docs/development.rst

```diff
@@ -799,134 +799,80 @@ Adding a new species
 To add a new species to `stdpopsim` several things are required:
 
 1. The genome definition
-2. Default species parameters
-3. A genetic map with local recombination rates (optional)
+2. Generation time estimate
+3. Mutation rate (per generation)
+4. Recombination rate (per generation)
+5. Characteristic population size
 
-Once you have these things the first step is to create a new file in the `catalog`
-directory named for the species (see `Naming conventions`_ for more details). All
-code described below should go in this file unless explicitly specified otherwise.
-
---------------------------
-Default species parameters
---------------------------
-
-Four default parameters are required to create a new species:
-
-1. Generation time estimate
-2. Mutation rate (per generation)
-3. Recombination rate (per generation)
-4. Characteristic population size
+A genetic map with local recombination rates is optional.
 
 These parameters should be based on what values might be drawn from a typical population
 as represented in the literature for that species. Consequently one or more citations for
 each value are expected and will be required for constructing the species object detailed
 below.
 
+Once you have these things the first step is to create a new file in the `catalog/`
+directory named for the species (see `Naming conventions`_ for more details). All
+code described below should go in this file unless explicitly specified otherwise.
```

Comment:
```suggestion
Once you have these things the first step is to create a new subdirectory of the `catalog/`
directory named for the species (see `Naming conventions`_ for more details). All
code described below should go in this directory unless explicitly specified otherwise.
```